docs/source/Install.rst

```diff
@@ -17,7 +17,7 @@ You should choose one method within each step and follow the instructions for th
 Step 1: Set up a virtual python environment
 ----------------------------------------------
 
-MapReader supports python versions 3.8 to 3.10.
+MapReader supports python versions 3.9+.
```

Comment:
You could add a markdown table that shows which versions of Python are supported by which versions of python like:

| Python | Mapreader |
| ---        | ---              |
| 3.8.x     | <= x.x.x.    |
| 3.9+     | All versions |

I remember a while ago we discussed having versions of readthedocs, so a user of mapreader working with an older mapreader version on python 3.8 who isn't ready to upgrade could see from this table that they'd want to look at the docs for the older mapreader version, which would still look as it used to.

Reply:
This is a good idea, I will do this now

Reply:
Added this to changelog